glean-core/python/glean/_loader.py

```diff
@@ -87,7 +92,15 @@ def _get_metric_object(metric: glean_parser.metrics.Metric) -> Any:
 
     glean_metric.__doc__ = metric.description
 
-    return glean_metric
+    yield name, glean_metric
+
+    # Events also need to define an enumeration
+    if metric.type == "event":
```

Comment:
I wish we could just use glean_parser for these kind of things :(

Reply:
Not sure I follow.  We *are* using glean_parser here.  This file is essentially what is required for any of the language generators (just generating objects, not code).  The parsing etc. still uses glean_parser underneath.

Reply:
Yup, sorry, let me clarify: I wish this was as simple as just creating a `jinja` template in glean_parser :)